pm2-explorer.json

```diff
@@ -0,0 +1,15 @@
+{
+        "apps": [{
+                "name": "explorer.app",
+                "script": "app.js",
+                "cwd": ".",
+                "pid_file": "./pids/explorer.app.pid",
+                "out_file": "./logs/explorer.app.log",
```

Comment:
Would be nice to use something like `pm2-logrotate` and limit log size, ppl have issues with logs fill entire disk space over and over again.